src/natcap/invest/ndr/ndr_core.pyx

```diff
@@ -54,6 +54,7 @@ def ndr_eff_calculation(
         suffix='.tif', prefix='flow_to_process',
         dir=os.path.dirname(effective_retention_path))
     os.close(fp)
+    algorithm = algorithm.lower()
```

Comment:
Ensuring that the algorithm is case-insensitive in these functions, since pre-processing doesn't happen here